dictionary/paymentsGetConsentsConsentId_v2.csv

```diff
@@ -50,7 +50,7 @@ O CNPJ será utilizado com 14 números e deverá ser informado sem pontos ou tra
 ";Texto;14;Obrigatório;^\d{11}$|^\d{14}$;;1;1;"";Não permitido;string;58764789000137;11
 /data/creditor/name;name;"Em caso de pessoa natural deve ser informado o nome completo do titular da conta do recebedor.  
 Em caso de pessoa jurídica deve ser informada a razão social ou o nome fantasia da conta do recebedor.
-";Texto;120;Obrigatório;^([A-Za-zÀ-ÖØ-öø-ÿ,.@:&*+_<>()!?/\\$%\d' -]+)$;;1;1;"";Não permitido;string;Marco Antonio de Brito;
+";Texto;120;Obrigatório;^([A-Za-zÀ-ÖØ-öø-ÿ' -]+)$;;1;1;"";Não permitido;string;Marco Antonio de Brito;
```

Comment:
Why this was removed? Seems that it was changed back to the old pattern.

Reply:
The data dictionary just is generate by the major version. 
the version 2.1.0 was changed to 3.0.0-beta.1.yml. 
In this case, it was neccessary buildded data dictionary again for rolback

Reply:
alright!